src/write.ts

```diff
@@ -145,6 +149,49 @@ function commentFooter(): string {
     return `This comment was automatically generated by [workflow](${actionUrl}) using [github-action-benchmark](https://github.com/marketplace/actions/continuous-benchmark).`;
 }
 
+function buildAnnotation(benchName: string, curSuite: Benchmark, prevSuite: Benchmark): string {
+    const lines = [
+        `Benchmark ${benchName}`,
+        '',
+        `Current: ${curSuite.commit.id} vs Previous: ${prevSuite.commit.id}`,
+        '',
+    ];
+
+    const benches = [];
+
+    for (const current of curSuite.benches) {
+        const prev = prevSuite.benches.find(i => i.name === current.name);
+
+        if (prev) {
+            const ratio = biggerIsBetter(curSuite.tool)
+                ? prev.value / current.value // e.g. current=100, prev=200
+                : current.value / prev.value;
+
+            benches.push([current.name, annVal(current), annVal(prev), floatStr(ratio)]);
+        } else {
+            benches.push([current.name, annVal(current), '', '']);
+        }
+    }
+
+    const size = [0, 0, 0, 0];
+
+    benches.forEach(row => {
+        size[0] = Math.max(row[0].length, size[0]);
+        size[1] = Math.max(row[1].length, size[1]);
+        size[2] = Math.max(row[2].length, size[2]);
+        size[3] = Math.max(row[3].length, size[3]);
+    });
+
+    return lines
+        .concat(
+            benches.map(row => {
+                const s = `${row[0].padEnd(size[0])} | ${row[1].padEnd(size[1])} | `;
+                return s + `${row[2].padEnd(size[2])} | ${row[3].padEnd(size[3])}`;
+            }),
+        )
+        .join('\n');
```

Comment:
I think including units (e.g. iter/ns) in footer would be good idea.

Reply:
I don't get it. They are added to the value themselves directly. Why would we need to add it to the footer too?

Reply:
ok.